src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -103,7 +107,7 @@ export async function makeTypescriptConfig(
         localRoot: config.codeRoot,
         remoteRoot: '/var/task',
         protocol: 'inspector',
-        skipFiles: ['/var/runtime/node_modules/**/*.js', '<node_internals>/**/*.js'],
+        skipFiles: ['/var/runtime/node_modules/**/*.js', '<node_internals>/**/*.js']
```

Comment:
can you disable whatever autoformatter is doing this?

Reply:
The reason it is disabled is the formatter will always append them even in places  that are weird like a array of size one like:
```
["string",]
```
vs
```
["string"]
```

Reply:
that seems like a reasonable tradeoff to me.

Reply:
> the formatter will always append them even in places that are weird like a array of size one like:

The prettier docs say it only applies to multiline code: https://prettier.io/docs/en/options.html#trailing-commas